pkg/kubecost/allocation.go

```diff
@@ -1347,13 +1336,13 @@ func (a *Allocation) getIdleKey(options *AllocationAggregationOptions) (string,
 		// Key allocations to ProviderId to match against node
 		idleKey = a.Properties.ProviderID
 		if idleKey == "" {
-			return idleKey, fmt.Errorf("ProviderId is not set")
+			return UnmountedSuffix, fmt.Errorf("ProviderId is not set")
 		}
 	} else {
 		// key the allocations by cluster id
 		idleKey = a.Properties.Cluster
 		if idleKey == "" {
-			return idleKey, fmt.Errorf("ClusterProp is not set")
+			return UnmountedSuffix, fmt.Errorf("ClusterProp is not set")
```

Comment:
Why is it the case that we should return "unmounted" and an error in all cases where an allocation is without a cluster ID?

Reply:
Same as above.